### DISCUSSION TITLE
Welcome to ragtacts Discussions!

### DISCUSSION THREAD

**Discussion by todd-kim** [constacts/ragtacts#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KetanPandya** [comment]
Hi Todd-kim,  I am a Clojure hobbyist  and a AI developer with experience with RAG using llamaIndex. I was keen to try out this great piece of work the team is developing but when I run tried the examples I am getting this issue. 
```
user=> user=> (-> (langchain/hub "rlm/rag-prompt")
    (prompt {:context "Ragtacts is an easy and powerful LLM library."
             :question "What is Ragtacts?"})
    ask
    last
    :ai)
;; "Ragtacts is an easy and powerful LLM library."
```
Execution error (IllegalArgumentException) at ragtacts.llm.base$eval15183$fn__15185/doInvoke (base.clj:41).
No value supplied for key: {:type :open-ai}

I have set up the OPENAI_API_KEY in my environment.

Please advise.

Thanks

Ketan
